<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmStart</class>
 <widget class="QDialog" name="frmStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>178</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Start Network Connection</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="optServer">
         <property name="text">
          <string>Start &amp;Server</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="optClient">
         <property name="text">
          <string>Start &amp;Client</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblServerIp">
         <property name="text">
          <string>Server IP:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtServerIp">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblPort">
         <property name="text">
          <string>Port:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtPort">
         <property name="text">
          <string>2333</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="chkAnyIp">
         <property name="text">
          <string>&amp;Use any available IPv4 address</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>S&amp;tart</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>C&amp;ancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="text">
          <string>E&amp;xit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>optServer</tabstop>
  <tabstop>optClient</tabstop>
  <tabstop>txtServerIp</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnExit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>frmStart</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
